wwstats.py

```diff
@@ -1,8 +1,9 @@
 import requests
 from achvlist import ACHV
 
-achv_names = [y['name'] for y in ACHV]
-total = len(ACHV)
+achv_names = {'en': [y['name'] for y in ACHV['en']], 'fa': [y['name'] for y in ACHV['fa']]}
+total = len(ACHV['en'])
+langs = ACHV.keys()
```

Comment:
I'd suggest changing this as follows:

```suggestion
langs = ACHV.keys()
total = len(ACHV['en'])
achv_names = {lang : [y['name'] for y in ACHV[lang]] for lang in langs}
```

This way, you don't need to explicitly mention all languages here once again and it will automatically work if more languages are added in the future :)